### PR TITLE
Menu options

### PR DESCRIPTION
**Pull request by fedeagripa**
Sidebar fixes (change to links so we prevent page reload)
Contact & About pages
Modal & TextArea commons

Pending
Contact action still doesnt do anything, has to be decided if the email is sent by react or the back

### CODE REVIEW

File: src/components/common/Modal.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React, { Component, PropTypes } from 'react';
+
+class Modal extends Component {
```
Comment (bernabe9) [inline src/components/common/Modal.js]: Use a stateless component for this component

File: src/components/common/Modal.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  }
+}
+
+Modal.propTypes = {
+  children: PropTypes.object,
+  closeModal: PropTypes.func.isRequired
+};
+
+export default Modal;
+
```
Comment (bernabe9) [inline src/components/common/Modal.js]: Remove this extra line

File: src/components/navbar/ContactForm.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import { Link } from 'react-router';
+import { routes } from '../../constants/routesPaths';
+import * as constraints from '../../utils/constraints';
+import Input from '../common/Input';
+import TextArea from '../common/TextArea';
+import smilies from '../../assets/smilies.svg';
+
+const ContactForm = ({ handleSubmit, error }) => (
+  <div>
+      <img src={smilies} className="smilies"/>
```
Comment (bernabe9) [inline src/components/navbar/ContactForm.js]: check the indentation

File: src/components/navbar/ContactNav.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import smilies from '../../assets/smilies.svg';
+import Modal from '../common/Modal';
+import ContactForm from './ContactForm';
+import { contact, closeModal } from '../../actions/navbarActions';
+import { routes } from '../../constants/routesPaths';
+import { Link } from 'react-router';
+
+class ContactNav extends Component {
```
Comment (bernabe9) [inline src/components/navbar/ContactNav.js]: Use a stateless component

File: src/components/navbar/ContactNav.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  render() {
+    return (
+      <div className="background-container">
+        <div className="top-container">
+          <Modal isOpen={true} closeModal={closeModal}>
```
Comment (bernabe9) [inline src/components/navbar/ContactNav.js]: When the prop is a boolean as `isOpen` is not necessary to declare the `true` value. Just `isOpen` should work.

```javascript
<Modal isOpen closeModal={closeModal} >
```

File: src/components/common/Modal.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import React, { Component, PropTypes } from 'react';
+
+class Modal extends Component {
+  render() {
+    return (
+      <div>
+        <div className="modalStyle">
+          <div className="thight-div">
+            <div className="force-right" onClick={ this.props.closeModal }> X </div>
```
Comment (bernabe9) [inline src/components/common/Modal.js]: Remove the space between the brackets `{` and `}`

**Review by bernabe9 — CHANGES_REQUESTED** [review]

**Review by felipegarcia92 — APPROVED** [review]
Approved because I don't have extra comments based on what @bernabe9 already commented. Fix those.

**Review by bernabe9 — APPROVED** [review]

**Pull request merged**